src/saveload/saveload.cpp

```diff
@@ -2582,6 +2582,7 @@ struct ZlibLoadFilter : LoadFilter {
 /** Filter using Zlib compression. */
 struct ZlibSaveFilter : SaveFilter {
 	z_stream z; ///< Stream state we are writing to.
+	byte fwrite_byte[MEMORY_CHUNK_SIZE]; ///< Buffer for writing to the file.
```

Comment:
Bikeshedding! Why `fwrite_byte` ? There are lots of bytes :) fwrite_buf seems more appropriate

Reply:
Also, is it worth just sticking this in the base class?

Reply:
It's a different size depending on the filter, so probably not.